src/CustomersList/Header/style.css

```diff
@@ -0,0 +1,36 @@
+body{
+    background-color:#ffffcc;
+    margin: 0%;
+    padding: 0%;
+}
+.head{
+    margin: 0 0 3px 0;
+    width: 100%;
+    height: 50px;
+    background-color: #5c145c;
+    display: flex;
+    justify-content: center;
+    position: relative;
+   
+}
+.head .inhead{
+   margin: 0;
+    width: 60%;
+    background-color: #5c145c;
+    color: white;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.inhead #username{
+    flex-grow: 2;
+    text-align: end;
+    padding-right: 25px;
+    
+}
+.mbt{  
```

Comment:
I told you already, mbt is not proper name. No developer will understand. Write properly.